packages/react-widgets/src/TimeList.js

```diff
@@ -41,9 +42,11 @@ function getBounds({ min, max, currentDate, value, preserveDate }) {
       max: end,
     }
   }
-
-  let start = dates.today()
-  let end = dates.tomorrow()
+  let start = dates.startOf(
+      dates.merge(value, min, currentDate),
+      'minutes'
+    )
+  let end = dates.merge(dates.add(start, 1, 'day'), max, currentDate)
```

Comment:
I think this breaks min/max settings, which _should_ take into account the day and compare it "now" and not allow time selection before or after the min/max datetimes